lib/excon/middlewares/redirect_follower.rb

```diff
@@ -17,7 +17,11 @@ def response_call(datum)
             params = datum.dup
             params.delete(:stack)
             params.delete(:connection)
-            params[:method] = :get if [301, 302, 303].include? response[:status]
+            if [301, 302, 303].include?(response[:status])
```

Comment:
I guess it was historically this way also, but from rereading the spec it isn't entirely clear to me that 303 should be in this list (though 301/302 seem correct). See: http://tools.ietf.org/html/rfc7231#section-6.4

By no means is this a problem with your pull, just something worth considering/discussing while we are looking at this part of the code base.